playbooks/aws/openshift-master/config.yml

```diff
@@ -30,11 +31,13 @@
   vars_files:
     - vars.yml
   roles:
-    - ../../../roles/base_os
-    - ../../../roles/repos
+    - base_os
+    - repos
     - {
-        role: ../../../roles/openshift_master,
-        oo_node_ips: "{{ hostvars['localhost'].oo_node_ips | default(['']) }}",
-        oo_bind_ip: "{{ hostvars[inventory_hostname].ansible_eth0.ipv4.address | default(['']) }}"
+        role: openshift,
+        openshift_roles: ['master'],
```

Comment:
This feels wrong. Having to specify a role inside of a role. This goes along with my other comment about logical roles.